### ISSUE TITLE
[BUG]: Dynasty Warriors 5 seams/lines on ground

### ISSUE THREAD

**Issue by Lolibox** [PCSX2/pcsx2#5218]
### Describe the Bug

This has been on the wiki for some time, i didnt see a github issue for it however.

A (usually) dark line that looks like the border of a group of triangles shows through the ground in many areas on most stages.

### Reproduction Steps

Any renderer or GS settings, i also tried interpreters for EE and VU, nothing seems to get rid of it.

### Expected Behavior

This line shouldnt appear.

### PCSX2 Revision

v1.7.2128

### Operating System

Windows 10 (64bit)

### CPU

i7-4790

### GPU

RX 570

### GS Window Screenshots

![pcsx2x64-avx2 2021-12-27 17-06-22](https://user-images.githubusercontent.com/49990481/147510170-9d0c3741-4469-44ff-bcb8-0a463868a17a.png)

### Logs & Dumps

[SLUS-21153 (6677B437).00.zip](https://github.com/PCSX2/pcsx2/files/7781922/SLUS-21153.6677B437.00.zip)
[gs_20211227171142.gs.zip](https://github.com/PCSX2/pcsx2/files/7781925/gs_20211227171142.gs.zip)
SS and Blockdump

**Comment by seta-san** [comment]
Set the rendering resolution to native and see if it goes away

**Comment by Lolibox** [comment]
>It is also independent of any resolution scaling
it does not.

**Comment by coornio** [comment]
Can confirm this happens in Dynasty Warriors 5: Empires as well. The only thing that appears to affect it is skipdraw, and judging by how parts of the ground seem to disappear using it, this might be some sort of ground texture blend border or lod system that's getting mis-aligned on a more foundational level. 

The Half-Pixel Offset (Vertex) hack has a minor effect on it but it's not a perfect fix, it only makes the line thinner. Round Sprite hack makes it worse in either setting. 

The only "good" fix I found was using Texture Offset, at least 800 in both fields. However, this results in a lot of mismatched textures everywhere else, such as dialog sprites.

**Comment by Buzzardsoul** [comment]
This patch is for Dynasty Warriors 5: Empires (which also suffers from the same issue)
SLES_540.95
patch=1,EE,001E57B8,word,00000000

**Comment by Mrlinkwii** [comment]
@Buzzardsoul  i think the pre-round pr fixes  it ( or did the last time i checked)

**Comment by Buzzardsoul** [comment]
> @Buzzardsoul i think the pre-round pr fixes it ( or did the last time i checked)

Yeah, I believe my patch might be just nopping a normal behavior that the gs is failing at.

**Comment by piau9000** [comment]
> Koei games with this issue have two memory addresses with the value of "3000000C". Changing one of them to "30000003" fixes the issue, but I don't know if this issue is a result of COP2/VU0, or GS. So I'll leave that to someone else to figure out.

How do you change these address? I have DW5 XL (US) and DW6 (US) and would like to try it myself.

**Comment by Buzzardsoul** [comment]
> > Koei games with this issue have two memory addresses with the value of "3000000C". Changing one of them to "30000003" fixes the issue, but I don't know if this issue is a result of COP2/VU0, or GS. So I'll leave that to someone else to figure out.
> 
> How do you change these address? I have DW5 XL (US) and DW6 (US) and would like to try it myself.

Something keeps writing to them, so patching the offsets won't work. You're gonna need to nop the instructions writing at them, and finding them is a luck-based hassle.
